---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* Ph.D in AI for Drug Discovery, Queen Mary University of London, Sept. 2023 - Present
  * Electronic Engineering and Computer Science
  * Target selection and prioritisation using prioritisation criteria and neuro-symbolic AI
  * Novel target selection and prioritisation through collaborative filtering between gene-disease association using graph and word embedding
  * Learning target prioritisation criteria with inductive logic programming for XAI

* M.Sc. in Bioinformatics, Wageningen University and Research, Sept. 2020 - Sept. 2022
  * Prediction of gene expression from local regulatory sequence using transformer and attention
  * Devised DNABERT-reg, a transformer encoder model to investigate genomic sequential features that affect gene expression level
  * Implemented interpretability techniques like attention, integrated-gradient, and SHAP to infer biologically relevant DNA motifs

* B.Sc. in GitHub, GitHub University, 2012

Work experience
======
* Feb. 2023 – Present: Junior Research Associate (Part-time)
  * Queen Mary University of London, London, United Kingdom
  * Use deep learning for simulation of population genomics data to infer parameters of recent evolution of Anopheles populations, including temporal changes in population size and migration rate among different geographical locations
  * Software development and packaging of tools for publication
  * Participated in team meetings and contributed to research discussions across the department of biological and behavioural science

* Apr. 2022 – Sept. 2022: Bioinformatics Genetics Intern
  * Dummen Orange, De Lier, Netherlands
  * Created imputation framework for polyploidy plants, boosting resolution for genomics selection accuracy, and decreasing sequencing cost for variant identification
  * Performed high-throughput processing of multi-level genotype variant data
  * Packaged imputation tools into Docker for standardized executable components for company’s genotyping pipeline

Skills
======
* Programming Languages: Python, R, Unix, SQL
* ML and DL techniques: Linear Regression, SVM, Random Forest, XGBoost, PyTorch, NLP, LLM (BERT), GAN, GNN
* Domains: AI, Multi-omics, Population Genetics, Natural Language Processing, Drug Discovery, causal inference
* Developer Tools: Git, Docker, Nextflow
* Interests: AWS, GenAI, app development

<!---
Publications
======
  <ul>{% for post in site.publications reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks reversed %}
    {% include archive-single-talk-cv.html  %}
  {% endfor %}</ul>
  
Teaching
======
  <ul>{% for post in site.teaching reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>

-->
  
Service and leadership
======
* Co-organizer of the Bi-weekly DERI lunch and learn seminar

Contact
======
* Email: pui.siu[AT]qmul.ac.uk
* LinkedIn: [Pui Chung Siu](https://www.linkedin.com/in/pui-chung-siu-6960b11a4/)
* GitHub: [jackysiupuichung](https://github.com/jackysiupuichung)
